charts/ccd-data-store-api/Chart.yaml

```diff
@@ -2,7 +2,7 @@ description: Helm chart for the HMCTS CCD Data Store
 name: ccd-data-store-api
 apiVersion: v1
 home: https://github.com/hmcts/ccd-data-store-api
-version: 1.5.1
+version: 1.5.2
```

Comment:
Considering the gravity of the changes, maybe 1.6?

Reply:
👍